### DISCUSSION TITLE
Why does the meta function run on the client as well?

### DISCUSSION THREAD

**Discussion by silvenon** [remix-run/remix#3036] [Q&A] [answered]
I noticed that the [`meta` function](https://remix.run/docs/en/v1/api/conventions#meta) runs on the client as well as on the server. Why not only on the server? Am I missing some obvious use case?

**Comment by sergiodxa** [comment, marked as answer]
When you do a client side navigation, the client has all the data to call meta and be able to change the title and meta tags for the new url

**Reply by silvenon** [reply]
Thanks, it's so obvious now! 😅
